tests/baselines/reference/parenthesizedContexualTyping1.types

```diff
@@ -220,11 +220,11 @@ var l = fun(((Math.random() < 0.5 ? ((x => x)) : ((x => undefined)))), ((x => x)
 >x => undefined : (x: number) => any
 >x : number
 >undefined : undefined
->((x => x)) : (x: number) => number
->(x => x) : (x: number) => number
->x => x : (x: number) => number
->x : number
->x : number
+>((x => x)) : (x: any) => any
+>(x => x) : (x: any) => any
+>x => x : (x: any) => any
+>x : any
+>x : any

```

Comment:
The change in the test output is related to the missing contextualMapper argument for parenthesized expressions (and just brings the test in line with the non-parenthesized case), but the widening of the function result from `undefined` to `any` is real culprit. We've got that issue captured in #241.